### PR TITLE
feat: add support to expand/collapse all log accordions at once

### PR DESCRIPTION
**Pull request by Elessar1802**
# Description

As the PR tittle suggests

Fixes https://github.com/devtron-labs/sprint-tasks/issues/901

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by AbhishekA1509 — APPROVED** [review]

**Comment by Elessar1802** [comment]
@AbhishekA1509 moved the css for log resize button to common

**Review by AbhishekA1509 — APPROVED** [review]

File: src/components/v2/appDetails/appDetails.type.ts [hunk 1]
```diff
@@ -446,6 +447,7 @@ export interface SyncErrorType {
 
 export interface SelectedResourceType {
     clusterId: number
+    clusterName: string
```
Comment (AbhishekA1509) [inline src/components/v2/appDetails/appDetails.type.ts:450]: Can we extend this

File: src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/terminal/Terminal.tsx [hunk 2]
```diff
@@ -268,13 +276,34 @@ export default function TerminalView({
             data-testid={dataTestId}
         >
             {renderConnectionStrip()}
+            {fullScreenView && (
+                <div className="w-100 flexbox dc__gap-6 dc__align-items-center px-12 py-4 terminal-wrapper__metadata">
+                    <ICDevtronLogo className="fcn-0 icon-dim-16" />
```
Comment (AbhishekA1509) [inline src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/terminal/Terminal.tsx:281]: Please add no shrink to this

**Review by AbhishekA1509 — CHANGES_REQUESTED** [review]
@abhibhaw In future we may have to give the existing features like resuming disconnected state and actions tile in full screen mode as well

**Review by AbhishekA1509 — DISMISSED** [review]

**Review by AbhishekA1509 — APPROVED** [review]

**Review by vivek-devtron — DISMISSED** [review]

**Comment by abhibhaw** [comment]
> @abhibhaw In future we may have to give the existing features like resuming disconnected state and actions tile in full screen mode as well

Agreed @AbhishekA1509

**Review by vivek-devtron — DISMISSED** [review]

**Review by vivek-devtron — APPROVED** [review]

**Pull request merged**